src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/RequestResolver.java

```diff
@@ -105,29 +116,28 @@
                         privilege,
                         origin,
                         innerRequest,
-                        headers,
+                        getThreadContextHeaders(),
```

Comment:
Are we sure that replacing "headers" with "getThreadContextHeaders()" will not cause any differences in behavior ?